src/classes/BDI_FieldMapping.cls

```diff
@@ -122,7 +122,7 @@ public virtual with sharing class BDI_FieldMapping {
             }
         }
 
-        // TODO: Complete when Template-level validation work item is in play
-        this.Is_Required = (this.Required == 'Yes' || !sourceFieldDescribe.isNillable() || !targetFieldDescribe.isNillable()) ? true : false;
+        this.Is_Source_Field_Required = !sourceFieldDescribe.isNillable() ? true : false;
+        this.Is_Required = !sourceFieldDescribe.isNillable() ? true : false;
```

Comment:
Nope! I was thinking it may be useful in the future when this ticket was still in flux. Blasted it for now.

https://github.com/SalesforceFoundation/Cumulus/pull/4849/commits/2edf43a8d63e69bd6940dbd9b57ae8194c51d606

Reply:
👍🏽